core/src/main/java/fr/sncf/osrd/conflicts/Conflicts.kt

```diff
@@ -33,10 +33,6 @@ class TrainRequirements(
 ) : SpacingTrainRequirement, RoutingTrainRequirement
 
 
-fun detectConflicts(trainRequirements: List<TrainRequirements>): List<Conflict> {
-    return incrementalConflictDetector(trainRequirements).checkConflicts()
-}
-
```

Comment:
Not sure why this moved. Usually, we try to keep the API near the top of the file, with more complex implementation details down below.